lms/services/lti_outcomes.py

```diff
@@ -147,7 +147,7 @@ def _get_body(cls, data):
             raise LTIOutcomesAPIError("Malformed LTI outcome response") from err
 
         if status != "success":
-            raise LTIOutcomesAPIError("LTI outcome request failed")
+            raise LTIOutcomesAPIError(explanation="LTI outcome request failed", response=data, details=data)
```

Comment:
Do we need to also pass `response=` here? I think that will change the logs a bit, but not exactly clear how.

Reply:
The `response` is already set by the caller of `_get_body` above when catching a `LTIOutcomesAPIError` exception, so it doesn't need to be set here.